src/meta/node/src/server.rs

```diff
@@ -862,8 +862,7 @@ pub async fn start_service_as_election_leader(
 
     #[cfg(not(madsim))]
     if let Some(dashboard_task) = dashboard_task {
-        // Join the task while ignoring the cancellation error.
-        let _ = dashboard_task.await;
+        dashboard_task.abort();
```

Comment:
The code seems no relation with `playground`? 👀

Reply:
Typo :') I meant to type dashboard